api/src/main/java/org/hawkular/apm/api/model/trace/Trace.java

```diff
@@ -72,18 +75,42 @@ public Trace() {
     }
 
     /**
-     * @return the id
+     * @return the traceId
+     */
+    public String getTraceId() {
+        return traceId;
+    }
+
+    /**
+     * This method sets the trace id. When the fragment and trace id
```

Comment:
If the field is not private, then it shouldn't be in the javadoc, as it can't be consumed outside of its own class. On this case, I think a class-level documentation is appropriate, as it's related to how `Traces` are related and compared.